Mention more contribution options in CONTRIBUTING.rst (#10202)
crate/crate: CONTRIBUTING.rst

This document is a guideline. Don't worry too much about getting everything
perfect. We are more than happy to work with you on your contribution.

Upvoting existing issues or reporting new issues it the easiest way to
contribute. But we also accept pull requests for the code and for the
documentation. For more information on how to work on the code or
documentation, see our `developer guide`_.

If you have any questions, please reach out via any of our `support channels`_.

Upvoting Issues
===============

An easy way to contribute is to upvote existing issues that are relevant to
you. This will give us a better idea what's important for you and other users.

Please avoid content-less ``+1`` comments but instead use the emoji reaction to

> upvote with a 👍. This allows to `sort issues by reaction`_ and doesn't

Reviewer: ```suggestion
upvote with a 👍. This allows people to `sort issues by reaction`_ and doesn't
```

spam the maintainers.

Tell us about your experience
=============================

> You don't have to create a detailed bug report, or request a new feature to

Reviewer: ```suggestion
You don't have to create a detailed bug report or request a new feature to
```

make a valuable contribution. Telling us about your experience with CrateDB is

> incredibly valuable for us as well. Please get in `contact`_ with us to tell us

Reviewer: ```suggestion
incredibly valuable as well. Please `get in touch`_ to tell us
```

this change might need a re-wrap

Reviewer: "get in touch" as a whole hyperlink turns it into an actionable CTA

See also: http://www.ericbmerritt.com/2011/09/21/commit-hygiene-and-git.html

.. _CLA: https://crate.io/community/contribute/agreements/

> .. _contact: https://crate.io/contact/

Reviewer: ```suggestion
.. _get in touch: https://crate.io/contact/
```
